src/answers-umd.js

```diff
@@ -293,9 +301,19 @@ class Answers {
         throw new Error('MasterSwitchApi determined the front-end should be disabled');
       }
       this._onReady();
+      this._searchOnLoad();
     });
   }
 
+  /**
+   * Calls each SearchBar's searchAfterAnswersOnReady().
```

Comment:
Give a little more detail here. I think we need to explain why we're calling things in this order.

Reply:
Updated the comment, how does it look?